client/src/data_layer_configs/lead_service_lines_by_parcel_config.ts

```diff
@@ -37,6 +38,16 @@ const LEGEND_VALUES = [
   },
 ];
 
+const createLegends = (): Map<GeographicLevel, LegendInfo> => {
+  const stateLegendInfo = {
```

Comment:
var is 'state' but key is County -- fix whichever is invalid

Reply:
Renamed